src/vs/workbench/api/browser/viewsExtensionPoint.ts

```diff
@@ -159,6 +161,10 @@ const viewDescriptor: IJSONSchema = {
 				localize('vscode.extension.contributes.view.initialState.hidden', "The view will not be shown in the view container, but will be discoverable through the views menu and other view entry points and can be un-hidden by the user."),
 				localize('vscode.extension.contributes.view.initialState.collapsed', "The view will show in the view container, but will be collapsed.")
 			]
+		},
+		size: {
+			type: 'number',
+			description: localize('vscode.extension.contributs.view.size', "The size of the view. Using a number will behave like the css 'flex' property, and the size will set the initial size when the view is first shown. In the sizebar, this is the height of the view."),
```

Comment:
Should "sizebar" become "side bar"? I think the doc convention is to write it as two words.